tests/unittests/config/test_cc_ubuntu_pro.py

```diff
@@ -443,8 +444,12 @@ class TestUbuntuProSchema:
                         " Use ``ubuntu_pro`` instead"
                     ),
                 ),
+                # If __version__ no longer exists on jsonschema, that means
```

Comment:
While this is a nice approach, and future-proof for when __version__ disappears. We still get a warning from pytest -[about the deprecation of __version__ in jsonschema](https://github.com/python-jsonschema/jsonschema/blob/main/jsonschema/__init__.py#L29-L34). Let's instead heed their deprecation warning advice.

WDYT about the following tested on focal lowest-supported 2.6.0 and py3 4.21:
```diff
--- a/tests/unittests/config/test_cc_ubuntu_pro.py
+++ b/tests/unittests/config/test_cc_ubuntu_pro.py
@@ -430,6 +430,12 @@ JSONSCHEMA_SKIP_REASON = (
 )
 
 
+def getJSONSchemaVersion() -> Version:
+    from importlib import metadata
+
+    return Version.from_str(metadata.version("jsonschema"))
+
+
 class TestUbuntuProSchema:
     @pytest.mark.parametrize(
         "config, expectation, skip_reason",
@@ -448,8 +454,7 @@ class TestUbuntuProSchema:
                 # we're using a high enough version of jsonschema to not need
                 # to skip this test.
                 JSONSCHEMA_SKIP_REASON
-                if Version.from_str(getattr(jsonschema, "__version__", "999"))
-                < Version(4)
+                if getJSONSchemaVersion() < Version(4)
                 else "",
                 id="deprecation_of_ubuntu_advantage_skip_old_json",
             ),
```

Reply:
We're still supporting 3.6 and 3.7, and this solution doesn't work there.

Reply:
Sure enough `New in version 3.8.` good catch. Hadn't thought through that as focal is 3.8